### ISSUE TITLE
[Link]: Inline option should be same color as text and underlined by default

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/carbon#3581]

**Comment by vpicone** [comment]
@dakahn curious if you have any insights here.

**Comment by mjabbink** [comment]
color alone cannot distinguish the inline link for sure. There must be an underline otherwise many cannot see the link at all making it pointless. Even in many cases that are not inline I recommend using the underline but optional is good. Example of no underline are anchor links and links in footer.

**Comment by snidersd** [comment]
@dakahn IMO this issue can be handled in the documentation.  According to the Carbon Component usage documentation, links should be used primarily as navigational elements, and the style section has examples of horizontal and vertical link groups which are considered navigational links that are visually evident from the page design and context, as described in https://www.w3.org/TR/WCAG20-TECHS/F73.html, the failure is when links are used within the text, such as links in a paragraph or items in a list where some are different than others and are presented in colored text. See https://www.w3.org/TR/WCAG20-TECHS/G182.html. 
The technique states that “Visual cues can take many forms including changes to the font style, the addition of underlines, bold, or italics, or changes to the font size.” Therefore an underline is one of a variety of ways to indicate the text is a link.

**Comment by vpicone** [comment]
Adding the additional requirement that the text-decoration should have the appropriate thickness in browsers that support defining this new property (Firefox, Safari, Safari IOS).

The feature is [not yet supported](https://caniuse.com/#feat=mdn-css_properties_text-decoration-thickness) in Edge or Chrome.

**Comment by mjabbink** [comment]
The Technique we want to use is underline as the very articles use in the above links. We do not want inline links to bold or Italic.

**Comment by vpicone** [comment]
@mjabbink per WCAG 2.0, an inline link is considered accessible if it has a 3:1 contrast ratio and an additional distinction is provided on hover.

<img width="693" alt="Screen Shot 2020-01-07 at 10 57 07 AM" src="https://user-images.githubusercontent.com/4078018/71913377-e5b71080-313c-11ea-8e97-b6bc2ac5ab8e.png">

**Comment by vpicone** [comment]
Design requirements:
- All links: blue by default, current color variant
- Regular links: underline on hover
- Inline links: always underlined
- Remove visited styles

**Comment by vpicone** [comment]
We went ahead and overrode/removed the `:visited` styles in the gatsby-theme per @mjabbink. Just a heads up that they're still very much a thing in Carbon.

**Comment by mjabbink** [comment]
👍
